### DISCUSSION TITLE
Conversation of EPS file to jpg file using Imagemagick

### DISCUSSION THREAD

**Discussion by Raji6707** [ImageMagick/ImageMagick#2804] [Development]
I am using dot net core 2.2 application and trying to convert the .EPS file to >JPEG file using ImageMagick(Magick.NET-Q8-ANYcpu)  using the below code.
using (MagickImage _image = new MagickImage())
                    {
                        string currentDir = Path.GetDirectoryName(Environment.CurrentDirectory);
                        MagickNET.SetGhostscriptDirectory($"{currentDir}\\gs");
                        _image.Read(fileBytes, settings);                    
                        _image.Write($"{newname}.jpg");                      
                    }
Also I have downloaded the ghostscript dll (gsdll64.dll) into the current working ditrectory of the api solution and from local machine I was successfully able to convert from .epd to .jpg files.

My application is deployed in azure and I have copied the ghostscript dlls (gsdll64.dll and gswin64c.exe) into the working directory of azure(inside wwwroot folder).But I am getting the below error.

"Internal Server Error from the custom middleware. FailedToExecuteCommand `\"D:/home/site/wwwroot/gs/gswin64c.exe\" -q -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 \"-sDEVICE=pngalpha\" -dTextAlphaBits=4 -dGraphicsAlphaBits=4 \"-r300x300\" -g390x390 -dFirstPage=1 -dLastPage=1 -dEPSCrop  \"-sOutputFile=D:/local/Temp/magick-3PGJlFRDww46T1BIqGnDJKGZax7IFlDC%d\" \"-fD:/local/Temp/magick-HDLYBdfbY1fMiS--df8GFg8f08v_Xbb2\" \"-fD:/local/Temp/magick-1VMVTsbqEz-PY5iX6GeyNJhed4CGnDQ7\"' (The system cannot find the file specified.\r\n) @ error/delegate.c/ExternalDelegateCommand/475 .

I saw somewhere like we have to edit the delegate.xml file as a solution .But I can't find the file in my solution.
Also I am using Ghostscript AGPL Release for downloading the ghostscript dll.
Please help me on solving the issue

**Comment by t246246** [comment]
I have no experience on Azure, but I suggest you check the permission of Folder.
Who run the gs command?  Is your D:/home/site/wwwroot/gs/ is readable? Is D:/local/Temp is both readable and writable for that user?  I don't know about Azure, but try to set the proper permission for all users first.

**Reply by dlemstra** [reply]
Did you manage to solve this @Raji6707?
